apps/web/components/App.tsx

```diff
@@ -259,11 +260,12 @@ const Component = ({
           </ul>
           <hr className="my-6" />
           <small className="leading-1 block text-gray-500">
-            Every app published on the Cal.com App Store is open source and thoroughly tested via peer
-            reviews. Nevertheless, Cal.com, Inc. does not endorse or certify these apps unless they are
-            published by Cal.com. If you encounter inappropriate content or behaviour please report it.
+            Every app published on the {WEBSITE_DOMAIN} App Store is open source and thoroughly tested via
+            peer peer reviews. Nevertheless, {WEBSITE_DOMAIN}, Inc. does not endorse or certify these apps
+            they are published by {WEBSITE_DOMAIN}. If you encounter inappropriate content or behaviour please
+            report it.
```

Comment:
this should stay Cal.com App Store because that is the Cal.com App Store and not the "ACME App Store", no? @zomars @baileypumfleet

Reply:
@PeerRich @zomars @baileypumfleet with regard to whiltelabel, this should be possible from my point of view